templates/back-office/packing-slip-content.hypr

```diff
@@ -98,7 +98,7 @@
                     <tbody>
                         {% for packagesItemDetails in packages.packingSlipItemDetails %}
```

Comment:
Please rename `packagesItemDetails` to be singular and remove trailing `s` from `packages`

e.g. `{% for packageItemDetail in package.packingSlipItemDetails %}`